rarity_checker.py

```diff
@@ -41,5 +41,5 @@ def main():
     check_number_rarity(args.records_db_path, args.number_to_check)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     main()
```

Comment:
wrap with `sys.exit(...)`

Reply:
`main` doesn't return anything and I'm too lazy to catch all the possible exceptions and return errors for them

Reply:
Continuing this thread in above conversation. What ever we decide there - do here.